src/init-helpers.ts

```diff
@@ -194,7 +194,7 @@ function prepareIframeNode(
     );
   }
 
-  iframe.setAttribute('allow', 'microphone;camera');
+  iframe.setAttribute('allow', 'microphone; camera; clipboard-read; clipboard-write');
```

Comment:
@tobiasz did you find this in a documentation somewhere? I can't seem to find anything about `clipboard-read` and `clipboard-write`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-allow
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy#directives